dj/models/query.py

```diff
@@ -67,7 +67,7 @@ class ColumnMetadata(SQLModel):
     """
 
     name: str
-    type: ColumnType
+    type: str
```

Comment:
Good catch, but wondering why no tests need to be adjusted.

Reply:
`ColumnType` inherits from `str` like the original `Enum` so it can be passed around. The idea was that it's just a `str` with this extra validation logic that an be used like the original `Enum` so the behavior would be intuitive or at least how I thought people would expect it to be